### ISSUE TITLE
FR: Support importing OTR key from KeySync

### ISSUE THREAD

**Issue by ghost** [redsolution/xabber-android#270]
Feature request: Support importing an OTR key file that was generated with KeySync ( https://guardianproject.info/apps/keysync/ ). KeySync is a desktop app from GuardianProject which supports several popular desktop IM clients.

This feature request is related to issue #68 , but narrower in scope.

**Comment by tyrion** [comment]
Exactly, it would be cool if someone could explain a workaround for this at least :)

**Comment by borkborkNFork** [comment]
:+1: This ability in any form would be very much appreciated.

**Comment by KonomiKitten** [comment]
It'd be nice to have an **export and import** of OTR keys in general, if KeySync is the best format to do this so be it.
